src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/DefaultRazorSemanticTokenInfoServiceTest.cs

```diff
@@ -55,31 +51,19 @@ public async Task GetSemanticTokens_CSharpBlock_HTML()
 
             var csharpTokens = new int[]
             {
-                14, 0, 3, RazorSemanticTokensLegend.CSharpKeyword, 0,
-                1, 0, 1, RazorSemanticTokensLegend.CSharpVariable, 0,
-                1, 0, 1, RazorSemanticTokensLegend.CSharpPunctuation, 0,
-                1, 0, 1, RazorSemanticTokensLegend.CSharpPunctuation, 0,
-                1, 0, 1, RazorSemanticTokensLegend.CSharpString, 0,
-                1, 0, 1, RazorSemanticTokensLegend.CSharpPunctuation, 0,
-                1, 0, 1, RazorSemanticTokensLegend.CSharpPunctuation, 0,
-                1, 0, 1, RazorSemanticTokensLegend.CSharpVariable, 0,
+                20, 4, 3, RazorSemanticTokensLegend.CSharpKeyword, 0, // var
```

Comment:
These values were previously inaccurate and have now been updated to the correct locations in the generated C# doc